### ISSUE TITLE
[Bug] Cloud Build ARGeoAnchors linking fail

### ISSUE THREAD

**Issue by thePostFuturist** [Unity-Technologies/arfoundation-samples#835]
Unity Cloud Build fails to build to what is seemingly tied to ARGeoAnchor. Figured it might belong here. ARCore Android builds fine on UCB and local. Snippet:

```
25769: ▸ Linking UnityFramework
25770: ❌;  Undefined symbols for architecture arm64
25771: > Symbol: _OBJC_CLASS_$_ARGeoAnchor
25772: > Referenced from: objc-class-ref in GeoAnchorsNativeInterop.o
25773: ❌;  ld: symbol(s) not found for architecture arm64
25774: ❌;  clang: error: linker command failed with exit code 1 (use -v to see invocation)
25775: ** ARCHIVE FAILED **
25776: The following build commands failed:
25777: 	Ld /BUILD_PATH/Library/Developer/Xcode/DerivedData/Unity-iPhone-evevnsgbtlkqxqbdiwzkqyvajlvu/Build/Intermediates.noindex/ArchiveIntermediates/Unity-iPhone/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/UnityFramework.framework/UnityFramework normal
25778: (1 failure)
25779: Exit status: 65
25780: Maybe the error shown is caused by using the wrong version of Xcode
25781: Found multiple versions of Xcode in '/APPLICATION_PATH/'
25782: Make sure you selected the right version for your project
25783: This build process was executed using '/APPLICATION_PATH/Xcode12_4_0.app'
25784: If you want to update your Xcode path, either
25785: - Specify the Xcode version in your Fastfile
25786: ▸ xcversion(version: "8.1") # Selects Xcode 8.1.0
25787: - Specify an absolute path to your Xcode installation in your Fastfile
25788: ▸ xcode_select "/APPLICATION_PATH/Xcode8.app"
25789: - Manually update the path using
25790: ▸ sudo xcode-select -s /APPLICATION_PATH/Xcode.app
25791: +-------------+-------------------------------+
25792: |              Build environment              |
25793: +-------------+-------------------------------+
25794: | xcode_path  | /APPLICATION_PATH/Xcode12_4_0.app |
25795: | gym_version | 2.182.0                       |
25796: | sdk         | iPhoneOS14.4.sdk              |
25797: +-------------+-------------------------------+
25798: ▸       objc-class-ref in GeoAnchorsNativeInterop.o
25799: ▸   "_OBJC_CLASS_$_ARGeoTrackingConfiguration", referenced from:
25800: ▸       objc-class-ref in GeoAnchorsNativeInterop.o
25801: ▸ ld: symbol(s) not found for architecture arm64
25802: ▸ clang: error: linker command failed with exit code 1 (use -v to see invocation)
25803: ⬆️  Check out the few lines of raw `xcodebuild` output above for potential hints on how to solve this error
25804: 📋  For the complete and more detailed error log, check the full log at:
25805: 📋  /BUILD_PATH/Library/Logs/gym/Simian-Unity-iPhone.log
```
Full:
[UCBFail.txt](https://github.com/Unity-Technologies/arfoundation-samples/files/6683556/UCBFail.txt)

Building on 2020.3.7f1.

**Comment by tdmowrer** [comment]
You probably need to add a dependency on the ARKit framework to the `GeoAnchorsNativeInterop.m` file in your project. You can do this from the file's inspector tab in the Editor.

**Comment by thePostFuturist** [comment]
That was totally it, thanks. BTW, it's also missing in the samples project for 2020.3.

**Comment by tdmowrer** [comment]
This should now be fixed in the samples by 5109082bf7ced4b815f5b5964a1ac6931d4fa95a
